src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -233,10 +233,10 @@ export const serviceDescription = appId => {
       return 'view your lab and test results';
 
     case widgetTypes.VIEW_APPOINTMENTS:
-      return 'view your appointments';
+      return 'view, schedule, or cancel your appointment online';
 
     case widgetTypes.SCHEDULE_APPOINTMENTS:
-      return 'schedule, reschedule, or cancel a VA appointment online';
+      return 'view, schedule, or cancel your appointment online';
```

Comment:
Not sure if this has since changed elsewhere, but I think Peggy's [copy in the issue thread](https://github.com/department-of-veterans-affairs/va.gov-team/issues/2525#issuecomment-594970731) has this as "view, schedule, or cancel **a VA** appointment online", not "your".  Otherwise LGTM!

Reply:
Oh, good catch. I didn't notice that it's different in her unauthenticated copy vs her authenticated copy. We have to make both the same. I'll check with her to figure out which one is better.